buildstockbatch/test/test_local.py

```diff
@@ -19,9 +19,8 @@
     "project_filename",
     [
         resstock_directory / "project_national" / "national_baseline.yml",
-        resstock_directory / "project_national" / "national_upgrades.yml",
         resstock_directory / "project_testing" / "testing_baseline.yml",
-        resstock_directory / "project_testing" / "testing_upgrades.yml",
+        # resstock_directory / "project_testing" / "testing_upgrades.yml",
```

Comment:
Why not replace with `sdr_upgrades_tmy3.yml`?

Reply:
The sdr_upgrades_tmy3.yml has a different output data structure so the test won't pass. I need to work on that.